config/client.lua

```diff
@@ -1,4 +1,15 @@
 return {
-    debugPoly = false,
     enableClient = true, -- disable to create your own client interface
+
+    --- called every frame when player is near the garage and there is a separate drop off marker
+    ---@param coords vector3
+    drawDropOffMarker = function(coords)
```

Comment:
functions in a config file doesn't seem like a good idea

Reply:
It's a pattern we've used in qbx_core before. Just a way to surface a particular function for editing and communicate that we expect people to edit it. It serves as a short cut for us to avoid having to support a large complex config of different basic features such as markers, 3D text, sprites, etc. And for our users who aren't able to navigate the code base and make modifications too easily, it isolates a simple function where they can see the parameters and feel safe that they won't break something else. Kind of like an interface. In fact, I have de ja vu that we discussed this exact thing in qbx_truckerjob PRs.

Reply:
I don't remember, but it's ok